e2e/jest-puppeteer.config.js

```diff
@@ -0,0 +1,44 @@
+const lodash = require('lodash');
+const puppeteer = require('puppeteer');
+const isHeadless = process.env.HEADLESS !== 'false';
+
+const NEW_CHROME_SWITCHES = [
```

Comment:
How did you decide on these flags - were these copied from elsewhere? Please add a comment.